app/src/androidTest/java/ch/epfl/sdp/blindly/profile_setup/TestProfileGenderMore.kt

```diff
@@ -59,18 +61,13 @@ class TestProfileGenderMore {
         Intents.init()
         onView(withId(R.id.text_p4_2)).perform(clearText(), typeText(NO_INPUT));
         closeSoftKeyboard();
-        val buttonContinue = Espresso.onView(withId(R.id.button_p4_2))
+        val buttonContinue = onView(withId(R.id.button_p4_2))
         buttonContinue.perform(click())
-        Espresso.onView(withId(R.id.warning1_p4_2))
-                .check(
-                        ViewAssertions.matches(
-                                ViewMatchers.withText(
-                                        Matchers.containsString(
-                                                ERROR_MESSAGE
-                                        )
-                                )
-                        )
-                );
+        onView(withId(R.id.warning1_p4_2))
+            .check(
+                ViewAssertions.matches(
+                    ViewMatchers.withText(
+                        Matchers.containsString(ERROR_MESSAGE))))
```

Comment:
Might be a good idea to import these functions directly so that you don't need to write `ViewAssertions`, `ViewMatchers` and `Matchers` anymore. (We did it in other test classes)

Reply:
I didn't write this piece of code (but I guess it considered it to be modified when I merged the main back into my branch).
But yes it would be nice if later when we refractor someone could clean up all the different imports in the tests

Reply:
In refactoring #97  I've done those imports for usual classes but not for the test classes. For normal classes it was more important in terms of usability and readability I believe, but it's possible to do in test classes as well. Though certainly not an urgent issue.